packages/features/ee/organizations/components/OrgPaymentHistory.tsx

```diff
@@ -0,0 +1,120 @@
+"use client";
+
+import { trpc } from "@calcom/trpc/react";
+import { Badge } from "@calcom/ui/components/badge";
+import { Button } from "@calcom/ui/components/button";
+import { PanelCard } from "@calcom/ui/components/card";
+import { SkeletonText } from "@calcom/ui/components/skeleton";
+
+export const OrgPaymentHistory = ({ orgId }: { orgId: number }) => {
+  const { data: billingData, isLoading } = trpc.viewer.organizations.adminGetBilling.useQuery({
+    id: orgId,
+  });
+
+  if (isLoading) {
+    return (
+      <PanelCard title="Payment History">
+        <div className="p-4 space-y-3">
+          <SkeletonText className="h-4 w-full" />
+          <SkeletonText className="h-4 w-full" />
+          <SkeletonText className="h-4 w-full" />
+        </div>
+      </PanelCard>
+    );
+  }
+
+  if (!billingData?.invoices || billingData.invoices.length === 0) {
+    return (
+      <PanelCard title="Payment History">
+        <div className="p-4 text-subtle text-sm">No payment history available for this organization.</div>
+      </PanelCard>
+    );
+  }
+
+  const formatCurrency = (amount: number, currency: string) => {
+    return new Intl.NumberFormat("en-US", {
```

Comment:
I think en-US makes the most sense for us to default to in admin tooling, Regardless if the bills are in your local currency in dev